protocol/rpcconsumer/rpcconsumer_server.go

```diff
@@ -208,8 +214,18 @@ func (rpccs *RPCConsumerServer) sendRelayToProvider(
 	// Make a channel for all providers to send responses
 	responses := make(chan *relayResponse, len(sessions))
 
+	// Set response timeout to 0
+	responseTimeout := 0 * time.Second
+
 	// Iterate over the sessions map
 	for providerPublicAddress, sessionInfo := range sessions {
+		// Calculate relay timeout
+		relayTimeout := extraRelayTimeout + common.GetTimePerCu(sessionInfo.Session.LatestRelayCu) + common.AverageWorldLatency
```

Comment:
this is not necessary as all sessionInfo.Session.LatestRelayCu are equal to chainMessage.GetServiceApi().ComputeUnits, so there is no need to find the maximum, we can set the timeout value outside of this function and remove a few lines of code